packages/zwave-js/src/lib/driver/UpdateConfig.ts

```diff
@@ -64,11 +65,38 @@ export async function installConfigUpdate(newVersion: string): Promise<void> {
 		);
 	}
 
+	const pkgFilepath = `${pak.cwd}/package.json`;
```

Comment:
Add this to the call to `await detectPackageManager({...})`:
```diff
+ setCwdToPackageRoot: true,
```
so `pak.cwd` actually contains the package root.